Tests/BuildTests/ClangTargetBuildDescriptionTests.swift

```diff
@@ -14,15 +14,40 @@ import Basics
 @testable import Build
 import PackageGraph
 import PackageModel
+import SPMBuildCore
 import SPMTestSupport
 import XCTest
 
 final class ClangTargetBuildDescriptionTests: XCTestCase {
     func testClangIndexStorePath() throws {
-        let targetDescription = try makeTargetBuildDescription()
+        let targetDescription = try makeTargetBuildDescription("test")
         XCTAssertTrue(try targetDescription.basicArguments().contains("-index-store-path"))
     }
 
+    func testSwiftCorelibsFoundationIncludeWorkaround() throws {
+        let macosParameters = mockBuildParameters(
+            toolchain: try UserToolchain.default,
```

Comment:
Do we need to use the real toolchain here? I wonder if the default `MockToolchain()` argument of `mockBuildParameters` would be sufficient?